repository_service_tuf/cli/admin/import_targets.py

```diff
@@ -0,0 +1,201 @@
+        rstuf_db_data = _parse_csv_data(csv_file, succinct_roles)
+        console.print(f"Import status: Importing {csv_file} data")
+        try:
+            db_client.execute(rstuf_table.insert(), rstuf_db_data)
+        except IntegrityError:
+            raise click.ClickException(
+                "Import status: ABORTED due duplicated targets. "
+                "CSV files must to have unique targets (path). "
+                "No data added to RSTUF DB."
+            )
+        console.print(f"Import status: {csv_file} imported")
+
+
+def _get_succinct_roles(metadata_url: str) -> SuccinctRoles:
+    response = request_server(metadata_url, "1.bin.json", Methods.get)
+    if response.status_code == 404:
+        raise click.ClickException("RSTUF Metadata Targets not found.")
+
+    json_data = json.loads(response.text)
+    targets: Metadata[Targets] = Metadata.from_dict(json_data)
+    if targets.signed.delegations is None:
+        raise click.ClickException("Failed to get Targets Delegations")
+
+    targets_delegations: Delegations = targets.signed.delegations
+
+    if targets_delegations.succinct_roles is None:
+        raise click.ClickException("Failed to get Targets succinct roles")
+
+    return targets_delegations.succinct_roles
+
+
+@admin.command()
+@click.option(
+    "-metadata-url",
+    required=True,
+    help="RSTUF Metadata URL i.e.: http://127.0.0.1 .",
+)
+@click.option(
+    "-db-uri",
+    required=True,
+    help="RSTUF DB URI. i.e.: postgresql://postgres:secret@127.0.0.1:5433",
+)
+@click.option(
+    "-csv",
+    required=True,
+    multiple=True,
+    help=(
+        "CSV file to import. Multiple -csv parameters are allowed. "
+        "See rstuf CLI guide for more details."
+    ),
+)
+@click.option(
+    "--skip-publish-targets",
+    is_flag=True,
+    help="Skip publishing targets in TUF Metadata.",
+)
+@click.pass_context
+def import_targets(
+    context: Any,
+    metadata_url: str,
+    db_uri: str,
+    csv: List[str],
+    skip_publish_targets: bool,
+):
+    """
+    Import targets to RSTUF from exported CSV file.
+    """
+    settings = context.obj["settings"]
+    server = settings.get("SERVER")
+    token = settings.get("TOKEN")
+    if server and token:
+        token_access_check = is_logged(server, token)
+        if token_access_check.state is False:
+            raise click.ClickException(
+                f"{str(token_access_check.data)}"
+                "\n\nTry re-login: 'Repository Service for TUF admin login'"
+            )
+    else:
+        raise click.ClickException("Login first. Run 'rstuf admin login'")
+
+    headers = get_headers(settings)
+
+    bs_response = request_server(
+        settings.SERVER, URL.bootstrap.value, Methods.get, headers=headers
+    )
+    if bs_response.status_code != 200:
+        raise click.ClickException(
+            f"Error {bs_response.status_code} {bs_response.text}"
+        )
```

Comment:
Here we are doing the same exact request and response check as in [here](https://github.com/vmware/repository-service-tuf-cli/blob/1320eb54b127d35c438bb7f66761906b2661efee/repository_service_tuf/helpers/api_client.py#L100-L105).
I am thinking maybe instead of a `get_headers` method, do we want a `get_bootstrap` or `valid_bootstrap` which is the same as `get_headers` (meaning it does all the validation) and if it succeeds it returns the bootstrap response.

Reply:
I think is the same what @MVrachev [mentioned above](https://github.com/vmware/repository-service-tuf-cli/pull/195#discussion_r1104658018)

Actually, `get_headers` returns the headers used for specific calls (other endpoints) after checking if the token is still valid (`is_logged`) and returns the headers formatted.

We can have another helper in the `api_client` to retrieve the bootstrap status. Because the status will be used in different behaviors.

Reply:
@MVrachev , @KAUTH
I was simulating this change here, it will endup in a lot of changes.
I think would be better do it in a different PR, otherwise it will deviate from the initial PR goal.

Reply:
> I was simulating this change here, it will endup in a lot of changes.
I think would be better do it in a different PR, otherwise it will deviate from the initial PR goal.

I am ok with that :+1: , there are indeed already a lot of changes

Reply:
Do we want to discuss this in a separate issue then?
If so, I think there are two discussions that it's good to be linked:
- this one
- and the one I started previously: https://github.com/vmware/repository-service-tuf-cli/pull/195#discussion_r1101454006

Reply:
I opened this issue with both references.
- https://github.com/vmware/repository-service-tuf-cli/issues/199